### ISSUE TITLE
[1.9.27] iPad Mini 3 / 9.3.4, iPhone 6 Plus / 10.0.2 - Application is crashing when user save the lomotif.

### ISSUE THREAD

**Issue by testershub** [Lomotif/lomotif-ios-issues#799]
Frequency: Every time

Action:
1: Launch the app.
2: Tap on "START" and select 1 video from "Camera roll".
3: Skip the music and proceed to "Create" screen.
4: Change the time frame position and save the lomotif.
5: Load the recently saved lomotif and make some changes.
6: Again save the lomotif.
7: Repeat step 5 and step 6 for 7 -10 times.
8: Check if application is crashing.

Expected result: Application should not crash.

Actual result: Application is crashing when user make changes to the existing lomotif more than 7-10 times and save it.

**Comment by testershub** [comment]
Refer following video for more details:
http://www.screencast.com/t/9aAaO3IQ

**Comment by hiteshbheda** [comment]
@xiuie I can reproduce this issue for build [1.9.27] on iPhone 6 / iOS 10.0.2

**I have repeated steps 5 and 6 for 12 to 15 times and then application crash.**

**Comment by xiuie** [comment]
Does this continue to happen after the first crash? i.e. every time you try to open it, it will crash? @testershub @hiteshbheda

**Comment by testershub** [comment]
@xiuie Yes, application crashes while opening the saved lomotif maximum times. But sometimes it crashes while making any changed to saved lomotif.

**Comment by hiteshbheda** [comment]
@xiuie Yes, application crash but it can only occurs when user change the lomotif and save. After performing 12 to 15 times, application crash once.
